test/UniswapV3Staker.spec.ts

```diff
@@ -160,10 +160,115 @@ describe('UniswapV3Staker', () => {
 
   describe('#endIncentive', async () => {
     describe('should fail if ', () => {
-      it('block.timestamp <= claim deadline')
-      it('incentive does not exist')
+      it('block.timestamp <= claim deadline', async () => {
+        const pool = await factory.getPool(
+          tokens[0].address,
+          tokens[1].address,
+          FeeAmount.MEDIUM
+        )
+        const rewardToken = tokens[0].address
+        const blockTime = await blockTimestamp()
+        const depositAmount = BNe18(1000)
+        const startTime = blockTime
+        const endTime = blockTime + 1000
+        const claimDeadline = blockTime + 2000
+
+        await tokens[0].approve(staker.address, BNe18(1000))
+
+        await staker.createIncentive(
+          rewardToken,
+          pool,
+          startTime,
+          endTime,
+          claimDeadline,
+          depositAmount
+        )
+
+        // Adjust the block.timestamp so it is before the claim deadline
+        await ethers.provider.send('evm_setNextBlockTimestamp', [
+          claimDeadline - 1,
+        ])
+
+        expect(
+          staker.endIncentive(
+            tokens[0].address,
+            pool,
+            startTime,
+            endTime,
+            claimDeadline
+          )
+        ).to.be.revertedWith('TIMESTAMP_LTE_CLAIMDEADLINE')
+      })
+      it('incentive does not exist', async () => {
```

Comment:
it's typical in ts to leave an extra line between blocks (unless it's the first one nested?)

Reply:
Thanks, I've pushed a fix for that now.